book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    ## has
+    @rentals.push(rental)
+    ## belongs to
+    rental.book = self
+
+    rental.person = self
+  end
```

Comment:
- [ ] Line 16 means you are assigning **_self_** to `rental.person`, which I think is abnormal. Please go through [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) from the example in a previous lesson. And try to implement this `add_rental` method correctly.